PsychENCODE/assay_TMTproteomics_schema.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id":"http://example.com/assay_TMTproteomics_schema.json",
+  "title": "PsychENCODE TMT Proteomics Assay Metadata Schema",
+  "description": "Validates PsychENCODE TMT proteomics assay metadata",
+  "type": "object",
+  "properties": {
+    "specimenID": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.3-dev/current_annotations_schemas/experimentalData.json#/definitions/specimenID"
+    },
+    "assay": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.3-dev/current_annotations_schemas/experimentalData.json#/definitions/assay"
+    },
+    "platform": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.3-dev/current_annotations_schemas/experimentalData.json#/definitions/platform"
+    },
+    "controlType": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.3-dev/reference_definitions_schemas/new_schema_refs.json#/definitions/controlType"
+    },
+    "batch": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.3-dev/reference_definitions_schemas/new_schema_refs.json#/definitions/batch"
+    },
+    "batchChannel": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.3-dev/reference_definitions_schemas/new_schema_refs.json#/definitions/batchChannel"
+    },
+    "TMTType": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.3-dev/reference_definitions_schemas/new_schema_refs.json#/definitions/TMTType"
+    },
+    "fragmentation": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.3-dev/reference_definitions_schemas/new_schema_refs.json#/definitions/fragmentation"
+    },
+    "resolution": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.3-dev/reference_definitions_schemas/new_schema_refs.json#/definitions/resolution"
+    },
+    "sampleID": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.3-dev/reference_definitions_schemas/new_schema_refs.json#/definitions/sampleID"
+    },
+    "multiconsensusStudyFileID": {
+      "$ref": "https://sage-bionetworks.github.io/sysbioDCCjsonschemas/releases/v0.0.3-dev/reference_definitions_schemas/new_schema_refs.json#/definitions/multiconsensusStudyFileID"
+    }
+  },
+  "required": [
```

Comment:
What is the impact of listing required columns here? This is not reflected in the dccvalidator: [see here](https://github.com/Sage-Bionetworks/dccvalidator/blob/master/config.yml#L121-L126)

I agree with adding `assay` but I'm not sure the others are required across all assay types.

Reply:
The required section in PsychENCODE schemas are probably not necessary since we have the dccvalidator and therefore don't have to use the schemas to validate the files.  I can take this out if you prefer.

Reply:
I don't have a preference.